valhalla/userrequests/templates/userrequests/userrequest_list.html

```diff
@@ -87,7 +90,7 @@
                   <i class="fa fa-fw fa-{{ ur.state|state_to_icon }}"></i>
                   {{ ur.state }}
                 </span></p>
-                <p><i class="fa fa-fw fa-calendar"></i> {{ ur.modified|naturaltime }}</p>
+                <p><i class="fa fa-fw fa-calendar"></i> <span id="tooltip-1" title="{{ ur.modified | naturaltime }}">{{ ur.modified }}</span></p>
```

Comment:
Can you give this span a class instead of an id? Id's are supposed to be unique, whereas classes can be repeated. This element is repeated for each item in the homepage.` class="tooltip"` would probably work fine.